### PR TITLE
Changed versions in installer script

### PR DESCRIPTION
**Pull request by ShivanshGahlot**

### CODE REVIEW

File: installer_scripts/install-yb-voyager [hunk 1]
```diff
@@ -11,11 +11,11 @@ ARGS_LINUX=$@
...
 # This needs to be changed to the latest debezium release tag
-VOYAGER_RELEASE_TAG=${VOYAGER_RELEASE_TAG:-"v1.6.5"}
-DEBEZIUM_VERSION=${DEBEZIUM_VERSION:-"2.3.3-1.6.5"}
+VOYAGER_RELEASE_TAG=${VOYAGER_RELEASE_TAG:-"v1.7.0-rc1"}
+DEBEZIUM_VERSION=${DEBEZIUM_VERSION:-"2.3.3-1.7.0-rc1"}
 
-# the hash corresponds to the yb-voyager/v1.6.5 tag
-YB_VOYAGER_GIT_HASH="05e9d3db3dd3c8dfda1f4167ac552c736563872f" 
+# the hash corresponds to the yb-voyager/v1.7.0-rc1 tag
+YB_VOYAGER_GIT_HASH="02213557c948480fc290b9379bb0ed3fbdbc9c8cdc6b13c0da53a17facd5a371"
```
Comment (priyanshi-yb) [inline installer_scripts/install-yb-voyager:18]: which version is this ? latest commit on `1.7-dev` - e0d47e87b1550a4bfa9c31446737167fa5fa7e70

**Review by priyanshi-yb — COMMENTED** [review]
Also make the change in `create_gather_assessment_metadata_dir`, remove the `YB_VOYAGER_GIT_HASH_TEMP`

File: installer_scripts/install-yb-voyager (hunk 1 above)
Comment (ShivanshGahlot) [reply installer_scripts/install-yb-voyager]: Fixed

**Review by priyanshi-yb — APPROVED** [review]
LGTM

**Review by sanyamsinghal — APPROVED** [review]

**Pull request merged**